modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/BaseSourceCheck.java

```diff
@@ -80,6 +80,23 @@ public int getWeight() {
 		return _weight;
 	}
 
+	public boolean hasParameterTypes(
+		String content, List<String> parameterList, String[] parameterTypes) {
```

Comment:
Can we convert `parameterList` to an array instead? Mixing a list and an array like this is a bit odd and `parameterList.get(i)` could be slow if the list is something like a `LinkedList`.

Reply:
Yes, we can!